### PR TITLE
FIF-18 add advanced search page

### PR DESCRIPTION
**Pull request by lalrg**

### CODE REVIEW

File: fixitfriday.ui/src/components/Survey/AdvancedSearchFilters.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+          />
+        </Col>
+        <Col sm={12} md={8} style={scrollableStyles}>
+          <Row>
+            <Col xs={12}>
+              <Form.Group controlId="SearchBar">
+                <Form.Control
+                  type="text"
+                  placeholder="Type student name or class name"
+                  aria-describedby="inputGroupPrepend"
```
Comment (kurtjohnson-HS) [inline fixitfriday.ui/src/components/Survey/AdvancedSearchFilters.tsx:57]: Where is this ID? Should we remove this?

Comment (lalrg) [reply fixitfriday.ui/src/components/Survey/AdvancedSearchFilters.tsx:57]: You are right... I removed it.

File: fixitfriday.ui/src/components/Survey/AdvancedSearch.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Row, Col } from 'react-bootstrap';
+
```
Comment (kurtjohnson-HS) [inline fixitfriday.ui/src/components/Survey/AdvancedSearch.tsx:3]: I like this blank line between dependencies and local imports. I need to start doing this.

File: fixitfriday.ui/src/components/Survey/AdvancedSearchFilters.tsx [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+type AdvancedSearchProps = {
+  onFilterUpdate?(filterCriteria: SurveyAdvancedSearch): void;
+};
+
+const scrollableStyles = {
+  maxHeight: '250px',
+  overflow: 'auto',
+};
+
+const AdvancedSearchFilters: React.FC<AdvancedSearchProps> = ({ onFilterUpdate }) => {
```
Comment (kurtjohnson-HS) [inline fixitfriday.ui/src/components/Survey/AdvancedSearchFilters.tsx:17]: It looks like AirBnb's guidance follows using `React.FC` over adding it as an import. I will start doing this as well.

**Pull request merged**